docs/07_maintainers_view.md

```diff
@@ -1,78 +1,70 @@
-# Maintainers View
+# Maintainers' Section
 
-So I want to document some of the views i have when maintaining NewPipe.
+These are some basic principles that I want maintainers to follow when maintaining NewPipe.
 
 
 ### Keep it Streamlined
-NewPipe is a Player for online videos on a smart phone, by means it is used
-for entertainment reason. This means it does not have to be some professional
+NewPipe is a media player for smartphones on the Android platform, thus it is intended to be used for entertainment. This means it does not have to be some professional
 application, and it does not have to be complicated to be used.
 However NewPipe might not focus on the casual user completely as there are
-many features that are a bit more "tecki" and may require some knowledge about
-technology, however all in all NewPipe should be easy to use, even for not teck
-guys.
-
-1. NewPipe does not have to be a air plane cockpit: __Don't add to much special
-  features__. If people want to do professionally things with Videos they
-  might use professional tools.
-2. __Design the UI so it does make sense to the user__. Try to make it comply with
+many features that are a bit more "techy" and may require some knowledge about
+code, however in essence NewPipe should be easy to use, even for your average Android user.
+
+1. __Don't add too many special
+  features.__ NewPipe does not have to be an air plane cockpit. Do not try to fill every single niche that might exist. If people want more advanced features, they
+  would use professional tools. If you add too much functionality, you add complexity, and complexity scares away the average user. Focus on NewPipe's scope as a **media player** for the end user, and only as such. 
+2. __Design the UI so that it is easy for the user__. Try to make it comply with
   [material design guidelines](https://material.io/design/guidelines-overview/).
-3. __Don't add to much features__: Think about the Betamax vs. VHS phenomena
-  or the Unix principle of having one program designed for one specific task:
-  If you add to much functionality you add complexity and this is not appealing
-  to the user. Focus on what NewPipe should be, and make it be only that.
+  
 
 ### Bugfixes
 
 ![kde_in_a_nutshell](img/kde_in_a_nutshell.jpg)]
 
-*Disclaimer: This is a meme maybe in real live it is different. Pleas no shit storm.*
+*Disclaimer: This is a meme. Don't take it personally.*
 
- __Always go for Bugfixes__, as the best application with the best features
+ __Always prioritize fixing bugs__, as the best application with the best features
```

Comment:
I think in American English it's `priorize`.

Reply:
The base noun is "priority", so the verb would be "prioritize". Google confirms the spelling.